t/admin/secrets.t

```diff
@@ -53,14 +53,14 @@ __DATA__
                         "prefix" : "apisix",
                         "token" : "apisix"
                     },
-                    "key": "/apisix/kms/vault/test1"
+                    "key": "/apisix/secrets/vault/test1"
```

Comment:
why we use `secrets` in admin API and use `secret` in etcd, according to the current style of APISIX, it should be standardised with `secrets`.